Lib/test/test_dbm.py

```diff
@@ -213,6 +220,27 @@ def test_whichdb_ndbm(self):
         for path in fnames:
             self.assertIsNone(self.dbm.whichdb(path))
 
+    @unittest.skipUnless(dbm_sqlite3, reason='Test requires dbm.sqlite3')
+    def test_whichdb_sqlite3(self):
+        # Test that databases created by dbm.sqlite3 are detected correctly.
```

Comment:
I'm not sure it's worth adding prefixes like `Test that`  since all tests test things :upside_down_face: 
```suggestion
        # Databases created by dbm.sqlite3 are detected correctly.
```